starboard/egl_and_gles/BUILD.gn

```diff
@@ -22,17 +22,26 @@
 # This decision is predicated on the value of the |gl_type| GN variable defined
 # in the configuration.gni for the current platform.
 
+import("//build/buildflag_header.gni")
+
 group("egl_and_gles") {
-  assert(gl_type != "none", "gl_type must be set to a valid value")
+  assert(gl_type == "angle" || gl_type == "system_gles2")
   public_deps = [ ":egl_and_gles_$gl_type" ]
 }
 
-group("egl_and_gles_system_gles2") {
-  # Use the system-provided implementation of GLES2.
+buildflag_header("buildflags") {
+  header = "buildflags.h"
+
+  _enable_angle = gl_type == "angle"
```

Comment:
No need to prepend an underscore here and below, it should already be scoped to this target